core/rio/rdfxml/pom.xml

```diff
@@ -61,5 +61,11 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.eclipse.rdf4j</groupId>
+            <artifactId>rdf4j-rio-turtle</artifactId>
```

Comment:
Could you explain a little bit why the rio-turtle dependency is needed here ? I would assume the code also compiles without it...

Reply:
Yes, it is compiling without it, but this was added automatically, we did not change anything in the rio-turtle files. We only added our code in the rio-util and helper files (RDFXMLPrettyWriter and BasicWriterSettings) only.

Reply:
Added automatically - that's really strange. Can I ask what editor you are using?

Reply:
IntelliJ IDEA Community Edition 2020.1.1 x64

Reply:
Strange, could you perhaps manually remove this, and see what happens ?

Reply:
This dependency had to be added back because of the `Class<?> arrangedWriterCls = Class.forName("org.eclipse.rdf4j.rio.turtle.ArrangedWriter");` line, without adding the dependency the code was not running.